network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_authors.html

```diff
@@ -7,28 +7,30 @@
 
   <p class="tw-text-sm my-2 tw-text-black">
     {% with authors_length=authors|length %}
-    {% spaceless %}
-    {% if show_full_info %}{% trans "By" %} {% endif %}
-    {% for author in authors %}
-      {% if not forloop.first %}{% if not forloop.last %}<span class="tw-font-extrabold">, </span>{% else %}<span> {% trans "and" %} </span>{% endif %}{% endif %}
-      {% if authors_length > 1 and forloop.last %}  {% endif %}
-        {% if blog_index %}
-          <a class="tw-font-extrabold tw-text-black" href="{% routablepageurl page=blog_index url_name='blog-author-detail' profile_slug=author.slug  %}">{{ author.name }}</a>
-        {% else %}
-          <span class="tw-font-extrabold">{{ author.name }}</span>
+      {% spaceless %}
+        {% if show_full_info %}
+          {% trans "By" %}
         {% endif %}
-    {% endfor %}
-    {% if show_full_info %}
-      | {% if blog_page.first_published_at %}{{ blog_page.first_published_at|date:"DATE_FORMAT" }}
-        {% else %}not published yet{% endif %}
-      {% with topic=page.specific.topics.first %}
-      {% if topic and blog_index %}
-        {% localized_version topic as localized_topic %}
-        | <a class="topic tw-text-inherit" href="{{blog_index.localized.url}}topic/{{topic.slug}}">{{ localized_topic }}</a>
-      {% endif %}
-      {% endwith %}
-    {% endif %}
-    {% endspaceless%}
+
+        {% for author in authors %}
+          {% if not forloop.first %}{% if not forloop.last %}<span class="tw-font-extrabold">, </span>{% else %}<span> {% trans "and" %} </span>{% endif %}{% endif %}
+          {% if authors_length > 1 and forloop.last %}  {% endif %}
+            {% if blog_index %}
+              <a class="tw-font-extrabold tw-text-black" href="{% routablepageurl page=blog_index url_name='blog-author-detail' profile_slug=author.slug  %}">{{ author.name }}</a>
```

Comment:
Double-space after `author.slug `

```suggestion
              <a class="tw-font-extrabold tw-text-black" href="{% routablepageurl page=blog_index url_name='blog-author-detail' profile_slug=author.slug %}">{{ author.name }}</a>
```